apps/crn-server/src/controllers/users.ts

```diff
@@ -109,9 +109,11 @@ export default class Users implements UserController {
     if (user.connections?.find(({ code }) => code === userId)) {
       return parseUserToResponse(user);
     }
+    const existingConnections =
+      user.connections?.filter(({ code }) => code !== welcomeCode) || [];
```

Comment:
Don't we want to keep the welcome code? 

I thought that was needed for some reason?

Reply:
i dont think so? why would we want to, do you think?